easybuild/tools/github.py

```diff
@@ -1059,3 +1060,23 @@ def validate_github_token(token, github_user):
         _log.info("GitHub token can be used for authenticated GitHub access, validation passed")
 
     return sanity_check and token_test
+
+
+def find_easybuild_eb():
+    """
+    Fetches the latest EasyBuild version eb file from GitHub
+    """
+    dev_repo = download_repo(GITHUB_EASYCONFIGS_REPO, branch='develop', account=GITHUB_EB_MAIN)
+    eb_parent_path = os.path.join(dev_repo, 'easybuild', 'easyconfigs', 'e', 'EasyBuild')
+    files = os.listdir(eb_parent_path)
+
+    # find most recent version
+    file_versions = []
+    for eb_file in files:
+        version = eb_file.split('-', 1)[1]
+        file_versions.append((eb_file, version.split('.eb', 1)[0]))
```

Comment:
wouldn't it be better to grab the `version =` line from the easyconfig file itself, rather than relying on the filename? would be cleaner imho...